control-plane-operator/hostedclusterconfigoperator/controllers/resources/ingress/reconcile.go

```diff
@@ -43,7 +43,7 @@ func ReconcileDefaultIngressController(ingressController *operatorv1.IngressCont
 	case hyperv1.AWSPlatform:
 		if useNLB {
 			ingressController.Spec.EndpointPublishingStrategy.LoadBalancer = &operatorv1.LoadBalancerStrategy{
-				Scope: operatorv1.ExternalLoadBalancer,
+				Scope: nlbScope,
```

Comment:
I think it actually makes sense to do this so we internal by default when aws topology is private cc @csrwng any concerns?

Reply:
No concerns from me